easybuild/easyconfigs/a/ABINIT/ABINIT-8.0.8-intel-2016a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '8.0.8'
+
+homepage = 'http://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy, charge density and
+ electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional
+ Theory (DFT), using pseudopotentials and a planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7b67d9ffc6200b3bcca0db12f7c69581']
+
+configopts = "--with-mpi-prefix=$EBROOTIMPI/intel64 --with-trio-flavor='etsf_io+netcdf' --with-dft=flavor='libxc' "
+configopts += '--with-netcdf-incs="-I$EBROOTNETCDF/include -I$EBROOTNETCDFMINFORTRAN/include" '
+configopts += '--with-netcdf-libs="-L$EBROOTNETCDF/lib -lnetcdf -L$EBROOTNETCDFMINFORTRAN/lib -lnetcdff" '
+configopts += '--with-libxc-incs="-I$EBROOTLIBXC/include" --with-libxc-libs="-L$EBROOTLIBXC/lib -lxc" '
+
+dependencies = [
+    ('libxc', '3.0.0'),
+    ('netCDF', '4.4.1'),
+    ('netCDF-Fortran', '4.4.4'),
```

Comment:
`netCDF-Fortran` depends on `netCDF 4.4.0` so you will need to change the dep above to match for Travis to pass it